airflow/hooks/dbapi.py

```diff
@@ -160,7 +161,25 @@ def get_first(self, sql, parameters=None):
                     cur.execute(sql)
                 return cur.fetchone()
 
-    def run(self, sql, autocommit=False, parameters=None, handler=None):
+    @staticmethod
+    def maybe_split_sql_string(sql: str) -> List[str]:
+        """
+        Splits strings consisting of multiple SQL expressions into an
+        TODO: do we need something more sophisticated?
+
+        :param sql: SQL string potentially consisting of multiple expressions
+        :return: list of individual expressions
+        """
+        splits = [s.strip() for s in re.split(";\\s*\r?\n", sql) if s.strip() != ""]
+        return splits
```

Comment:
I think we are using sqlparse for some of the hooks; we should use it here as well. It is a pretty small dependency and more reliable than whatever we come up with.

Reply:
I need some suggestions on where to put `sqlparse` dependency.
It seems that it should be in every provider that uses `DbApiHook`

Reply:
Which is the same case as with `pandas`

Reply:
Basically none of the providers will be able to use it for quite some time anyway as this is part of Airlfow, so really this is not a feature of any of the providers, but it is feature of Airlfow - you will not use it by installing a specific version of a specific providers but by migrating to Airlfow 2.4.